File: Cython/fib.cpp
#include <stdio.h>

int calc_fib(int n){
    double a=0.0, b=1.0, c;
    for(int i=0;i<n;i++){
        c=a+b;
        a=b;
        b=c;
    }
   return a;
}

int main(){
    int n;
    printf("input n s.t. fin(n)\n");
    scanf("%d",&n);
    double a=calc_fib(n);
    printf("%f\n",a);
}

File: Cython/fib.py

def calc_fib(n):
    a,b=0.0,1.0
    for i in range(n):
        a,b=b,a+b
    return a
def main():
    n=int(input('input n s.t. fib(n): '))
    res=calc_fib(n)
    print(res)
if __name__ == '__main__':
    main()